src/openforms/scss/components/admin/_logic-trigger.scss

```diff
@@ -1,12 +1,29 @@
-@import '~microscope-sass/lib/responsive';
+@import '~microscope-sass/lib/bem';
 @import '~microscope-sass/lib/color';
+@import '~microscope-sass/lib/responsive';
 
 $logic-line-breakpoint: 1800px !default;
 
+$logic-trigger-border: solid 1px #eee;
+
+.logic-trigger-container {
+  padding: 1em;
```

Comment:
Shouldn't the variables in the `src/openforms/ui/static/ui/scss/_settings.scss` file be used instead of hardcoded values?

Reply:
yes but no? :P those variables aren't very portable/exact science - spacing ratio's heavily depend on their parent context. Besides that, the `UI` scss stuff is still leftovers from the prototype that should be revisited at some point.

And I'd rather use CSS variables than sass variables for this kind of stuff so you can tackle it with theming more easily (e.g. a theme with larger spacing for people that desire it vs. a compact theme for people who want to see as much as possible on a single page)